theme/templates/resource-landing-page/citation.html

```diff
@@ -11,551 +11,550 @@
+                            <button {% if not cm.can_be_public_or_discoverable %} disabled {% endif %}
+                                    id="btn-public" data-toggle="tooltip" data-placement="auto"
+                                    title='Can be viewed and downloaded by anyone.' type="submit"
+                                    class="btn btn-default">Public
+                            </button>
+                        </form>
+                    {% endif %}
+
+                    {# -------- DISCOVERABLE -------- #}
+                    {% if not cm.raccess.public and discoverable %}
+                        <button id="btn-discoverable" data-toggle="tooltip" data-placement="auto"
+                                title='Metadata is public but data is protected.' type="button"
+                                class="btn btn-default active">Discoverable
+                        </button>
+                    {% else %}
+                        <form class="pull-left"
+                              action="/hsapi/_internal/{{ cm.short_id }}/change-permissions/"
+                              method="POST">
+                            <input name="t" type="hidden" value="make_discoverable"/>
+                            {% csrf_token %}
+                            <button {% if not cm.can_be_public_or_discoverable %} disabled {% endif %}
+                                    id="btn-discoverable" data-toggle="tooltip" data-placement="auto"
+                                    title='Metadata is public but data is protected.' type="submit"
+                                    class="btn btn-default">Discoverable
+                            </button>
+                        </form>
+                    {% endif %}
+
+                    {# -------- PRIVATE -------- #}
+                    {% if not cm.raccess.public and not discoverable %}
+                        <button id="btn-private" data-toggle="tooltip" data-placement="auto"
+                                title='Can be viewed and downloaded only by designated users or research groups.'
+                                type="button" class="btn btn-default active">Private
+                        </button>
+                    {% else %}
+                        <form class="pull-left"
+                              action="/hsapi/_internal/{{ cm.short_id }}/change-permissions/"
+                              method="POST">
+                            <input name="t" type="hidden" value="make_private"/>
+                            {% csrf_token %}
+                            <button id="btn-private" data-toggle="tooltip" data-placement="auto"
+                                    title='Can be viewed and downloaded only by designated users or research groups.' type="submit"
+                                    class="btn btn-default">Private
+                            </button>
+                        </form>
+                    {% endif %}
+                </div>
```

Comment:
@pkdash @mauriel133 Just a quick note that the changes made is actually very simple as shown below, but since I added indentation to make code blocks much more readable, the changes marked by github look much more than the actual changes. To facilitate code review, I am copying the actual change snippet below for easy reference:
```
<div id="user-permission-msg">
            {% if is_owner_user %}
                <div class="clear-fix"><i>You are the owner of this resource.</i></div>
            {% elif is_edit_user %}
                <div class="clear-fix"><i>You have been given specific permission to {% if cm.raccess.published %}view{% else %}edit{% endif %} this resource.</i></div>
            {% elif is_view_user %}
                 <div class="clear-fix"><i>You have been given specific permission to view this resource.</i></div>
            {% endif %}
        </div>
        {% if not cm.raccess.published and is_owner_user or cm.raccess.shareable and is_view_user or cm.raccess.shareable and is_edit_user %}
```